src/annotations/components/AddAnnotationOverlay.tsx

```diff
@@ -16,11 +16,8 @@ export const AddAnnotationOverlay: FC = () => {
   const {createAnnotation, startTime} = useSelector(getOverlayParams)
 
   const handleSubmit = (modifiedAnnotation): void => {
-    const formIsValid = true
```

Comment:
Let's not remove this here or in the other place. Instead, the fix is not to remove this, but to hook up `formIsValid` to a real validation function: https://github.com/influxdata/ui/issues/980